content.md

```diff
@@ -364,70 +365,87 @@ my_contact_book=# \dt
+
+Rails DB provides a graphical user interface (GUI) to our database, to make it easier for developers to quickly see all the data in the database. The Rails DB GUI is accessible in our app at the URL `/rails/db`.
 
 - Start your web server with `bin/server`.
 - Open the live app preview. You should see the default Rails homepage, since we haven't defined a root route yet.
-- Manually navigate to the URL `/rails/db`. The route, controller, action, and view for this URL is provided for us by the Rails DB gem. But you should have seen a 404 "no route matches" error! Read on to find out why.
+- Manually navigate to the URL `/rails/db`. You will see a 404 page.
 
-<aside>
+Why is it a 404? The Rails DB gem is installed, and it provides the Route, Controller, Action, and View for this page; but, for security reasons, we need to take a couple of extra steps to enable to gem.
 
-As an alternative to manually navigating to `/rails/db`, you can also click the "Database" link from our "Developer Toolbar" on the right side of the page:
+<aside>
+As an alternative to manually navigating to `/rails/db`, you can also click the "Database" link from the "Developer Toolbar" on the right side of the page:
 
 ![Dev toolbar gif](/assets/dev-toolbar.gif)
 
-That 🛠️ icon for the "Dev Toolbar" is in all of our Rails applications to provide some handy links during development.
+You'll see that 🛠️ icon in all of our Rails apps to provide some handy links during development.
 </aside>
 
 #### Add environment variables to access Rails DB
 
-If you tried to visit `/rails/db` just now, you should have seen a 404 "no route matches" error. Since `/rails/db` gives full access to CRUD in our database, we pre-configured our Rails projects to block access without two [secure environment variables for authorizing visitors](https://learn.firstdraft.com/lessons/52-storing-credentials-securely#loading-environment-variables-in-ruby-on-rails) (**Note**, as the guide linked there states, since we are in Rails, you _don't_ need to add the `dotenv` gem and `bundle install`; it's already included out-of-the-box!). You will need to:
+If you tried to visit `/rails/db` just now, you should have seen a 404 "no route matches" error. The GUI located at `/rails/db` gives full access to CRUD our database, which is dangerous! We don't want just anyone to be able to access it.
+
+So, we pre-configured our Rails projects to block access unless two [environment variables](https://learn.firstdraft.com/lessons/52-storing-credentials-securely#loading-environment-variables-in-ruby-on-rails) are present.
 
-- Cancel your server with `CTRL` + `C`
-- Create a `.env` file in the root of your project
-- Add the `RAILS_DB_USERNAME` and `RAILS_DB_PASSWORD` environment variables to the `.env` file (copy and paste from below, but _change the \*\*\*\* values to something unique and secure_)
+Let's add those environment variables now:
+
+- Shut down your web server with <kbd>control</kbd> + <kbd>c</kbd>.
```

Comment:
```suggestion
- Shut down your web server with <kbd>Ctrl</kbd> + <kbd>c</kbd>.
```

Perhaps better / more standard key name? Though I'm staring at my two keyboards and the mac laptop does say `control` whereas my external keyboard says `Ctrl`.

But I think `Ctrl` is pretty standard and used elsewhere in our lessons.

Reply:
Yes you're right, we use `Ctrl` everywhere else. (And capital for letters.)